install_sedumi.m

```diff
@@ -156,6 +163,16 @@ function install_sedumi( varargin )
     IS64BIT  = ~ISOCTAVE & strcmp(COMPUTER(end-1:end),'64');
     flags = {};
     libs = {};
+    
+    if ~isempty(openblas_path)
+      if ~isdir(openblas_path)
```

Comment:
`isdir` is looking for directories on the search path https://www.mathworks.com/help/matlab/ref/isdir.html, same in Octave.  Maybe:

```
if exist (openblas_path, 'dir') == 7
```

Reply:
Does it behave the same in Octave 4.0.3? As far as I can remember, in Octave 4.0.3 it was mostly just a wrapper around a line similar to yours. If that's no longer the case, I am fine with the change.

Reply:
I can confirm that `isdir.m` in Octave 4.0.3 is defined as follows:

```
function retval = isdir (f)
  if (nargin != 1)
    print_usage ("isdir");
  endif

  ## Exist returns an integer but isdir should return a logical.
  retval = (exist (f, "dir") == 7);

endfunction
```

Thus, your line is equivalent. Change applied.